modules/govuk_env_sync/manifests/task.pp

```diff
@@ -45,13 +45,14 @@
   $database,
   $url,
   $path,
+  $ensure = 'absent',
```

Comment:
Why did you go for a default of `absent`?

Reply:
I want it to break in case it gets ill configured/ I miss the configuration update. Mainly with focus on having to turn off integration sync tonight..